ops/c/src/mpi/ops_mpi_rt_support_sycl.cpp

```diff
@@ -618,9 +618,9 @@ void ops_dat_fetch_data_slab_memspace(ops_dat dat, int part, char *data, int *ra
       prod *= target->size[d];
     }
     ops_kernel_descriptor *desc = ops_dat_deep_copy_core(target, dat, range);
-    desc->name = "ops_internal_copy_device";
-    desc->device = 1;
-    desc->function = ops_internal_copy_device;
+    strcpy(desc->name, "ops_internal_copy_device\0");
```

Comment:
Why do we need the null character here?

Reply:
name is a char*

Reply:
yes, but as far as I am aware, when you have a cost char* string, like "ops_internal_copy_device", it implicitly comes with a terminating \0 char at the end. Please check and confirm. If it really does need the \0 at the end, then we'll need that everywhere else in the code too.

Reply:
yes i checked. It is not required.
removing it